docs/usage/shoot_high_availability_best_practices.md

```diff
@@ -390,9 +390,13 @@ This configures vertical pod autoscaling in Gardener-managed clusters. See [abov
+If a node fails to come up, the node group (worker pool in that zone) will go into "back-off", at first 5m, then [exponentially longer](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/utils/backoff/exponential_backoff.go#L77-L82) until the maximum of 30m is reached. The "back-off" is reset after 3 hours. This in turn means, that nodes must be first considered `Unknown`, which happens when `spec.kubernetes.kubeControllerManager.nodeMonitorPeriod.nodeMonitorGracePeriod` lapses (e.g. at the beginning of a zone outage). Then they must either remain in this state until `spec.provider.workers.machineControllerManager.machineHealthTimeout` lapses for them to be recreated, which will fail in the unhealthy zone, or `spec.kubernetes.kubeAPIServer.defaultUnreachableTolerationSeconds` lapses for the pods to be evicted (usually faster than node replacements, depending on your configuration), which will trigger the cluster autoscaler to create more capacity, but very likely in the same zone as it tries to balance its node groups at first, which will fail in the unhealthy zone. It will be considered failed only when `maxNodeProvisionTime` lapses (usually close to `spec.provider.workers.machineControllerManager.machineCreationTimeout`) and only then put the node group into "back-off" and not retry for 5m (at first and then exponentially longer). Only then you can expect new node capacity to be brought up somewhere else.
+
+During the time of ongoing node provisioning (before a node group goes into "back-off"), the cluster autoscaler may have "virtually scheduled" pending pods onto those new upcoming nodes and will not reevaluate these pods anymore unless the node provisioning fails (which will fail during a zone outage, but the cluster autoscaler cannot know that and will therefore reevaluate its decision only after it has given up on the new nodes).
+
+It's critical to keep that in mind and accommodate for it. If you have already capacity up and running, the reaction time is usually much faster with leases (whatever you set) or endpoints (`spec.kubernetes.kubeControllerManager.nodeMonitorPeriod.nodeMonitorGracePeriod`), but if you depend on new/fresh capacity, the above should inform you how long you will have to wait for it and for how long pods might be pending (because capacity is generally missing and pending pods may have been "virtually scheduled" to new nodes that won't come up until the node group goes eventually into "back-off" and nodes in the healthy zones come up).
```

Comment:
```suggestion
It's critical to keep that in mind and accommodate for it. If you have already capacity up and running, the reaction time is usually much faster with leases (whatever you set) or endpoints (`spec.kubernetes.kubeControllerManager.nodeMonitorGracePeriod`), but if you depend on new/fresh capacity, the above should inform you how long you will have to wait for it and for how long pods might be pending (because capacity is generally missing and pending pods may have been "virtually scheduled" to new nodes that won't come up until the node group goes eventually into "back-off" and nodes in the healthy zones come up).
```

Reply:
Well... nobody (including of course myself) noticed that in the original document. I fixed it now. Thanks.